presto-docs/src/main/sphinx/release/release-0.218.rst

```diff
@@ -2,6 +2,12 @@
 Release 0.218
 =============
 
+.. warning::
+
+    This release has the potential to produce incorrect results for three way joins involving a ``FULL OUTER JOIN`` and one subquery of
+    the ``FULL OUTER JOIN`` has a ``GROUP BY`` on the join key and some expression that could be evaluated to a constant. For details, please
+    refer to :issue:`12577`.
+
 General Changes
```

Comment:
I rephrased this a bit, hope it is still correct.

```
This release may produce incorrect results for three way joins where one of the joins is a ``FULL OUTER JOIN`` and one subquery of the ``FULL OUTER JOIN`` has a ``GROUP BY`` on the join key and an expression that can be evaluated to a constant value. For details, please refer to :issue:`12577`.
```

Reply:
It's the first join (in order of running). But "one of the joins" is not less inaccurate than "involving".